src/python/pants/backend/python/framework/django/detect_apps_test.py

```diff
@@ -142,8 +127,3 @@ def test_works_with_python38(rule_runner: RuleRunner) -> None:
 @skip_unless_python39_present
 def test_works_with_python39(rule_runner: RuleRunner) -> None:
     assert_apps_detected(rule_runner, constraints1="CPython==3.9.*")
-
-
-@skip_unless_all_pythons_present(PY_27, PY_39)
```

Comment:
Presumably we still want to test partitioning by ICs, just not these ICs?

Reply:
Makes sense. I can restore the test but switch away from Python 2.7 as the IC.

Reply:
Done.